src/Microsoft.AspNet.Hosting/WebApplicationConfiguration.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.AspNet.Hosting
+{
+    public class WebApplicationConfiguration
+    {
+        public static readonly string ApplicationKey = "app";
+        public static readonly string DetailedErrorsKey = "detailedErrors";
+        public static readonly string EnvironmentKey = "environment";
+        public static readonly string ServerKey = "server";
+        public static readonly string WebRootKey = "webroot";
+        public static readonly string CaptureStartupErrorsKey = "captureStartupErrors";
+
+        public static readonly string HostingJsonFile = "hosting.json";
+        public static readonly string EnvironmentVariablesPrefix = "ASPNET_";
+
+        public static IConfiguration GetDefault()
+        {
+            return GetDefault(args: null);
```

Comment:
Not sure if that's an ok default. Should we just assume the command line args for any process should be used as config? Feels wrong.